modules/rosa-creating-cluster.adoc

```diff
@@ -96,3 +107,17 @@ If installation fails or the `State` field does not change to `ready` after 40 m
 ----
 $ rosa logs install --cluster=<cluster_name> --watch
 ----
+
+. Enable ROSA to create cluster-specific roles. The roles created include the relevant prefix for the cluster name.
```

Comment:
The same questions:

1. I am confused, the account-roles should be used for sts cluster, may I know why put it in regular rosa cluster creation doc?
2. Is `manual` mode by default in command expected here for user to  run the commands by themselves? I think use `--mode auto -y` should be better in the command

Reply:
`--mode auto -y` is the easiest way to create these, but not all customers will have the necessary permissions to run them like this. I suggest removing the `--mode` entirely and let the CLI drop the user into interactive mode, where they will be able to select the right option for their use case.

Reply:
I have now moved the STS-related steps to the [Creating a cluster for using STS](https://deploy-preview-34960--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa-sts-creating-cluster.html#rosa-sts-creating-cluster_rosa-sts-creating-cluster) assembly.

I have also removed the `--mode` option from the commands and included a description for each that refers to the interactive mode options. I have also included a note under each to state that it is possible to use the `--mode` option to specify the mode in the command directly.